### PR TITLE
Fixed #25161 -- Encouraged users to install the latest release in docs

### PR DESCRIPTION
**Pull request by alasdairnicol**
Not sure the wording is perfect, but it's a starting point. Feel free to improve.

### CODE REVIEW

File: docs/intro/install.txt [hunk 1]
```diff
@@ -57,12 +57,13 @@ You've got three easy options to install Django:
   who have operating systems that distribute Django.
 
 * :ref:`Install an official release <installing-official-release>`. This
-  is the best approach for users who want a stable version number and aren't
-  concerned about running a slightly older version of Django.
+  is the best approach for users who want the latest release.
```
Comment (timgraham) [inline docs/intro/install.txt]: latest stable release?

File: docs/intro/install.txt [hunk 2]
```diff
@@ -57,12 +57,13 @@ You've got three easy options to install Django:
...
 
 * :ref:`Install the latest development version
-  <installing-development-version>`. This is best for users who want the
+  <installing-development-version>`. This is good for users who want the
```
Comment (timgraham) [inline docs/intro/install.txt]: enthusiast users

File: docs/intro/install.txt [hunk 3]
```diff
@@ -57,12 +57,13 @@ You've got three easy options to install Django:
...
   latest-and-greatest features and aren't afraid of running brand-new code.
+  You might encounter new bugs in the development version. Reporting
```
Comment (timgraham) [inline docs/intro/install.txt]: , but reporting them helps...

File: docs/intro/install.txt [hunk 4]
```diff
@@ -57,12 +57,13 @@ You've got three easy options to install Django:
...
+  these helps the development of Django.
```
Comment (timgraham) [inline docs/intro/install.txt]: Also, releases of third-party packages are less likely to be compatible with the development version than the latest stable release.

**Comment by alasdairnicol** [comment]
Thanks @timgraham, excellent suggestions, I've updated the patch.

File: docs/intro/install.txt [hunk 5]
```diff
@@ -57,12 +57,15 @@ You've got three easy options to install Django:
   who have operating systems that distribute Django.
 
 * :ref:`Install an official release <installing-official-release>`. This
-  is the best approach for users who want a stable version number and aren't
-  concerned about running a slightly older version of Django.
+  is the best approach for users who want the latest stable release.
```
Comment (carljm) [inline docs/intro/install.txt]: This feels a bit circular. "The latest official release is the best approach for users who want the latest stable release." Ok, so we've learned that "official -> stable" :-)

I would be tempted to simplify even further: "This is the best approach for most users."

**Comment by carljm** [comment]
:+1:

File: docs/intro/install.txt (hunk 5 above)
Comment (alasdairnicol) [inline docs/intro/install.txt]: @carljm I agree, I'll update the patch.

**Comment by timgraham** [comment]
merged in 600fb7c51b22bc0d9138f580bb5e5abcfedc76ee, thanks!

**Pull request closed without merging**